README.md

```diff
@@ -155,28 +155,43 @@ The ⓒ mark means that [`@uppy/companion`](https://uppy.io/docs/companion), a s
   <img src="https://saucelabs.com/browser-matrix/transloadit-uppy.svg" alt="Sauce Test Status"/>
 </a>
 
-We aim to support IE11 and recent versions of Safari, Edge, Chrome, Firefox and Opera.
+We aim to support recent versions of Safari, Edge, Chrome, Firefox and Opera.
 
-We still run end-to-end tests with IE10, but we are not actively supporting it or fixing visual / minor issues.
+We still run end-to-end tests with IE11, but we are not actively supporting it or fixing visual / minor issues.
```

Comment:
Does something still need to happen in order to run e2e tests against IE11 if we just removed the polyfills?

Reply:
Seems we just removed e2e tests for IE11 so this should change